Dockerfile

```diff
@@ -37,11 +37,6 @@ RUN mkdir /srv/ttn-lorawan/public/blob
 
 VOLUME ["/srv/ttn-lorawan/public/blob"]
 
-ENV TTN_LW_BLOB_LOCAL_DIRECTORY=/srv/ttn-lorawan/public/blob \
-    TTN_LW_IS_DATABASE_URI=postgres://root@cockroach:26257/ttn_lorawan?sslmode=disable \
-    TTN_LW_REDIS_ADDRESS=redis:6379 \
-    TTN_LW_HEALTHCHECK_URL=http://localhost:1885/healthz/live
```

Comment:
We still need ENV for the HEALTHCHECK URL, which needs to be configurable in case someone changes the port number, or sets auth on that endpoint.